File: src/1619.mean-of-array-after-removing-some-elements.1210/solution.ts
function trimMean (arr: number[]): number {
    arr.sort((a, b) => a - b);
    const fivePervent = arr.length / 20;
    let sum = 0;
    for (let i = fivePervent; i < arr.length - fivePervent; i++) {
        sum += arr[i];
    }

    return sum / (arr.length * 0.9);
}

File: src/1619.mean-of-array-after-removing-some-elements.1210/solution.java
class Solution {
    public double trimMean(int[] arr) {
        Arrays.sort(arr);
        int R = arr.length/20;
        int sum = 0;
        for(int i=R;i<arr.length-R;i++){
            sum += arr[i];            
        }
        return (double) sum/(arr.length-2*R);
    }
}